src/resources/vue/pages/index.vue

```diff
@@ -1,11 +1,40 @@
 <template>
```

Comment:
Moet dit al op de homepage staan? Ik denk van niet omdat we eerst alle componenten maken